packages/common/src/ContractInteractor.ts

```diff
@@ -1158,6 +1158,19 @@ calculateTransactionMaxPossibleGas: result: ${result}
     return await this.relayRegistrar.getRelayRegistrationMaxAge()
   }
 
+  async getRelayInfo (relayHubAddress: string, relayManagerAddress: string): Promise<{
```

Comment:
why would you need the params? don't you have them in the ContractInteractor ?

Reply:
?

Reply:
We only have hub in ContractInteractor, not manager address.

Reply:
right the manager is per-relay. the relayHubAddress is redundant.